web/packages/teleport/src/components/SlidingSidePanel/InfoGuideSidePanel/InfoGuideSidePanel.tsx

```diff
@@ -117,7 +125,7 @@ export const InfoTitle = styled(H3)`
   margin-top: ${p => p.theme.space[3]}px;
 `;
 
-export const InfoParagraph = styled(P)`
+export const InfoParagraph = styled(Box)`
```

Comment:
this was required since `lists` can appear inside this block, er... now that i'm reading this again maybe I should rename `InfoParagraph` as `InfoBlock`?